src/oemof/solph/flows/_non_convex_flow.py

```diff
@@ -373,6 +373,13 @@ def _create(self, group=None):
                 if g[2].nonconvex.minimum_uptime is not None
             ]
         )
+        self.MAXUPTIMEFLOWS = Set(
```

Comment:
Must be placed close to https://github.com/oemof/oemof-solph/blob/b6dc604527fb865ee67744534b2ad789bb670a64/src/oemof/solph/flows/_non_convex_flow_block.py#L231 as the _non_convex_flow.py was refactored into _non_convex_flow_block.py